src/Target.cpp

```diff
@@ -446,6 +446,27 @@ bool Target::supported() const {
     return !bad;
 }
 
+bool Target::supports_device_api(DeviceAPI api) const {
+    switch (api) {
+    case DeviceAPI::Parent:
```

Comment:
IMHO this should either get its own 'return true' or a comment about 'explicit fall-thru'